pkg/webui/console/views/device-general-settings/network-server-form/index.js

```diff
@@ -201,9 +216,18 @@ const NetworkServerForm = React.memo(props => {
         phyVersionRef.current = phyVersion
 
         getMacSettings(freqPlan, phyVersion)
+        dispatch(getBandsList(bandId, phyVersion))
```

Comment:
If we do the request for bands list via `<RequireRequest />` in the `device-general-settings/index.js` don't we run the risk of the whole section not being rendered, considering that the device request might return an empty `version_ids` and the request for bands list requires the `band_id` as argument. I think it would make sense to wrap the overriding data rate fields in a `<RequireRequest />`, no?

Reply:
You can do conditional requests if you pass an `action` function into `<RequireRequest />`. Either that or dynamically compose the action array based on whether `version_ids` are present in the device.